modules/engines/engine-xsjs/src/test/resources/META-INF/dirigible/test/xsk/http/http.xsjs

```diff
@@ -1,15 +1,12 @@
 let http = $.net.http;
 var assertTrue = require('utils/assert').assertTrue;
-let client = new http.Client();
-let request = new http.Request(http.GET, "/");
-let destination = new http.Destination()
 
-destination.host="https://services.odata.org";
-destination.pathPrefix="/V4/Northwind/Northwind.svc/";
+let client = new http.Client();
+var request = new $.net.http.Request($.net.http.GET, "/V4/Northwind/Northwind.svc/");
 
```

Comment:
this change essentially removes testing the destination support.
Better to adjust the `XSKApiSuiteTest` and provide a custom destination loader
See https://sap.github.io/cloud-sdk/docs/java/features/connectivity/sdk-connectivity-destination-service#using-custom-destination-loaders